test/e2e-test-application/package.json

```diff
@@ -59,17 +59,16 @@
     "babel-preset-minify": "^0.5.0",
     "codelyzer": "^4.4.1",
     "concurrently": "^4.1.0",
-    "cypress": "^3.8.2",
-    "cypress-plugin-retries": "^1.2.0",
```

Comment:
cypress-plugin-retries not applicable anymore with cypress 5.0.0, [reference](https://docs.cypress.io/guides/references/migration-guide.html#Tests-retries)